macros/HTMLRef.ejs

```diff
@@ -115,8 +115,8 @@ var resultAPI = [];
   <%    } 
     } %>
     <% if (env.slug.includes("/HTML/Element/input")) { %>
-   <li data-default-state="open"><a href="/<%=env.locale%>/docs/Web/HTML/Element/input#Form_<input>_types"><%-text['Input_types']%></a>
-    <%-template("ListSubpagesForSidebar", ['/en-US/docs/Web/HTML/Element/input'])%></li>
+   <li><details open><summary><%-text['Input_types']%></summary>
```

Comment:
On https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio, the existing sidebar has a collapsed ``HTML Elements`` section, while it is a plain list for the new sidebar:

<img width="602" alt="pr 789 htmlref audio" src="https://user-images.githubusercontent.com/286017/45366579-c047e000-b5a4-11e8-8b6f-e31b87b6d3fa.png">

The extra ``<input>`` menu looks good on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox, with some minor whitespace changes and "link" styling for ``<input> types`` section text:

<img width="614" alt="pr 789 htmlref input checkbox" src="https://user-images.githubusercontent.com/286017/45366514-9e4e5d80-b5a4-11e8-969c-68040777ef73.png">

Reply:
HTML Elements is now an expandable list but, there is an additional 20px bottom padding on the list elements here that is not on the other sidebars. It makes the sidebar very tall, and the spacing between elements, is to tall.

Reply:
@schalkneethling The 20px padding is on all first level li's `.quick-links li`, and is on production like that as well. You don't notice it all the time because we apparently have a negative margin on all `.toggle` li's that closes the  gap. You can, however, see it here for example: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies

All that said, I think we should reduce it and remove the negative margin because A) it's a lot and B) it leads to some weirdness like in John's screenshot above where a toggle link is right after an first level li.

So long story short: on it. :)

Reply:
All set on Kuma.